spec/services/irs_attempts_api/tracker_spec.rb

```diff
@@ -5,13 +5,29 @@
     allow(IdentityConfig.store).to receive(:irs_attempt_api_enabled).and_return(
       irs_attempt_api_enabled,
     )
+    allow(request).to receive(:headers).and_return(
+      "User-Agent" => 'bananas/1.0',
+      "Referer" => 'google.com',
```

Comment:
I meant to clean these up to something less goofy before pushing this up, but alas...